### ISSUE TITLE
One moment... Invalid Password.

### ISSUE THREAD

**Issue by JoooostB** [d8ahazard/Phlex#155]
Dear d8ahazard,

Running a clean CentOS 7 VM here, PHP up and running with Sockets, Curl & OpenSSL enabled. Whenever I try to login though, I get nothing more than a pop-up that my password is invalid. But it really isn't. Phlex has permission in it's folder, and the logs say the following:
`[Sun, 15 Oct 2017 15:41:49 +0000] [INFO] [api.php::checkSignIn] - Trying to sign into Plex.tv as JoooostB.
[Sun, 15 Oct 2017 15:41:49 +0000] [DEBUG] [util.php::signIn] - No token or not signed in, signing into Plex.
`
The Phlex_error.log is empty. What's going wrong here?

Thanks in advance

**Comment by d8ahazard** [comment]
Anything else in the logs aside from that line?

I'd suggest checking for PHP error logs as described here:

https://www.centos.org/forums/viewtopic.php?t=28047

**Comment by muppet3000** [comment]
@d8ahazard I know you've closed this but I've just managed to run into the same issue on a CentOS 7 VM and I've also managed to reproduce it inside a docker container. 
I've tracked it down to an issue in the util.php file where it signs in, the curl library that's being used (JMathai\PhpMultiCurl\MultiCurl) generates a curl request that results in a 400 response (bad request) which I'm guessing means that the syntax of the curl is somehow being generated incorrectly on CentOS 7.
I've managed to build a Fedora based docker container with all of the same steps that I used on CentOS 7 and it logs in absolutely fine, it also works around the problem with the current docker container that you provide not supporting hosting behind a reverse proxy (https://github.com/d8ahazard/docker-phlex/issues/10). I haven't actually tried using it yet though, however once I do it may be a viable alternative to the current docker container.
